yarn.lock

```diff
@@ -4433,6 +4433,26 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask/permission-controller@npm:^4.0.0":
```

Comment:
Nit: Perhaps we could add a resolution to remove the duplicate copy of the permission controller. The only uses at the moment are in dead code.